packages/codemods/package.json

```diff
@@ -54,7 +54,6 @@
     "@types/fs-extra": "11.0.4",
     "@types/jest": "29.5.8",
     "@types/jscodeshift": "0.11.6",
-    "@types/prettier": "2.7.3",
```

Comment:
Prettier provides its own types now. See the deprecation warning on https://www.npmjs.com/package/@types/prettier.